frontend/src/components/authchooser/index.tsx

```diff
@@ -261,9 +261,14 @@ export function PureAuthChooser({
             <Box alignItems="center" textAlign="center">
               <Box m={2}>
                 <Empty>
-                  {t('Failed to get authentication information: {{ errorMessage }}', {
-                    errorMessage: error!.message,
-                  })}
+                  {error && error.message === 'Bad Gateway'
+                    ? t(
+                        "Please make sure the Kubernetes cluster is running and accessible, Recieved '{{ errorMessage }}' when trying to connect",
```

Comment:
Please add these to the other locales too (so joaquimrocha can review/edit them more easily).